src/config/2021.json

```diff
@@ -350,6 +360,14 @@
       "github": "cqueern",
       "twitter": "httpsecheaders"
     },
+    "cdixon83": {
+      "name": "Carlie Dixon",
+      "teams": [
+        "reviewers"
+      ],
+      "avatar_url": "https://avatars.githubusercontent.com/u/76669481?v=4&s=200",
+      "github": "cdixon83"
+    },
```

Comment:
@cdixon83 FYI I've added your name but since it's not on your GitHub profile let me know if you'd rather go by something else